fake-pab/default.nix

```diff
@@ -1,14 +1,15 @@
 { pkgs, gitignore-nix, haskell, webCommon, webCommonMarlowe, buildPursPackage, buildNodeModules, filterNpm, marlowe-dashboard }:
 let
+  nativeOnly = pkgs.lib.meta.addMetaAttrs { platforms = with pkgs.lib.platforms; [ linux darwin ]; };
```

Comment:
We define this in a few places, should we put it somewhere common and pass it in?